benchmarks/b12x_vs_cutlass_moe_sweep.py

```
@@ -0,0 +1,347 @@
+            x_q, x_sf = fp4_quantize(x_bf16, cutlass_w["a1_gs"])
+
+            def run_cutlass(x, te, rw, w1q, w2q, o):
+                return cutlass_fused_moe(
+                    x,
+                    te.to(torch.int),
+                    rw,
+                    w1q.contiguous().view(torch.long),
+                    w2q.contiguous().view(torch.long),
+                    torch.bfloat16,
+                    quant_scales=cutlass_w["quant_scales"],
+                    input_sf=x_sf,
```

Ruff (0.15.14)

[warning] Function definition does not bind loop variable `x_sf`

(B023)